windows/src/README.md

```diff
@@ -29,7 +29,7 @@ For local development you do not need to perform a release build so these are op
 
 ## Building Keyman for Windows and Keyman Developer
 
-1. Start 'x64_x86 Cross Tools Command Prompt for VS 2017'.
+1. Start 'Developer Command Prompt for VS 2019'.
```

Comment:
Why not "x64_x86 Cross Tools Command Prompt for VS 2019"? I don't know the difference but I can see when I run the "x64_x86" prompt the file vcvarsall.bat is run.

Reply:
We don't really need the Cross Tools Command Prompt. That's for running x64-specific tooling on an x86 system, I think. The normal "Developer Command Prompt" works fine.